lemminx-liberty/src/main/java/io/openliberty/tools/langserver/lemminx/services/LibertyWorkspace.java

```diff
@@ -158,6 +160,16 @@ public void setContainerName(String containerName) {
         this.containerName = containerName;
     }
 
+    public String getContainerType() {
+        return containerType;
+    }
+
+    public void setContainerType(String containerType) {
+        if (containerType != null && !containerType.isEmpty()) {
+            this.containerType = containerType;
+        }
```

Comment:
If the containerType was set to podman, and then subsequently null was passed in here because they used an older version of the plugin, we should default back to docker. I think there should be an ` else { this.containerType = "docker"; }`.